frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTemplateSection.tsx

```diff
@@ -76,34 +107,252 @@ const ServingRuntimeTemplateSection: React.FC<ServingRuntimeTemplateSectionProps
+              }
+              onClick={() => {
+                if (
+                  getServingRuntimeNameFromTemplate(template) !== data.servingRuntimeTemplateName ||
+                  data.scope !== SERVING_RUNTIME_SCOPE.Global
+                ) {
+                  setData(
+                    'servingRuntimeTemplateName',
+                    getServingRuntimeNameFromTemplate(template),
+                  );
+                  setData('scope', SERVING_RUNTIME_SCOPE.Global);
+
+                  setServingRuntimeDisplayName(getServingRuntimeDisplayNameFromTemplate(template));
+                  // Reset model framework selection when changing the template in KServe modal only
+                  if (resetModelFormat) {
+                    resetModelFormat();
+                  }
+                }
+              }}
+              icon={<GlobalIcon />}
+            >
+              <Split>
+                <SplitItem isFilled>
+                  <Truncate content={getServingRuntimeDisplayNameFromTemplate(template)} />
+                </SplitItem>
+                <SplitItem>
+                  {compatibleIdentifiers?.some((identifier) =>
+                    isCompatibleWithIdentifier(identifier, template.objects[0]),
+                  ) && (
+                    <Label color="blue">
+                      Compatible with{' '}
+                      {isHardwareProfilesAvailable ? 'hardware profile' : 'accelerator'}
+                    </Label>
+                  )}
+                </SplitItem>
+              </Split>
+            </MenuItem>
+          ))}
+      </MenuGroup>
+    </>
+  );
+
   return (
+    // TODO: need to add popover from this PR: https://github.com/opendatahub-io/odh-dashboard/pull/3955
     <FormGroup label="Serving runtime" fieldId="serving-runtime-template-selection" isRequired>
-      <SimpleSelect
-        isFullWidth
-        isDisabled={isEditing}
-        dataTestId="serving-runtime-template-selection"
-        aria-label="Select a template"
-        options={options}
-        placeholder={
-          isEditing || filteredTemplates.length === 0
-            ? data.servingRuntimeTemplateName
-            : 'Select one'
-        }
-        toggleProps={{ id: 'serving-runtime-template-selection' }}
-        value={data.servingRuntimeTemplateName}
-        onChange={(name) => {
-          // Avoid onChange function is called twice
-          // In KServe modal, it would set the model framework field to empty if there is only one option
-          if (name !== data.servingRuntimeTemplateName) {
-            setData('servingRuntimeTemplateName', name);
-            // Reset model framework selection when changing the template in KServe modal only
-            if (resetModelFormat) {
-              resetModelFormat();
-            }
-          }
-        }}
-        popperProps={{ appendTo: 'inline' }}
-      />
+      {isProjectScoped && projectSpecificTemplates && !projectSpecificTemplates[1] ? (
```

Comment:
this gets a little long with nested ternaries and is kind of hard for me to follow. i wonder if we can move it out of `return` and use more `if (..) { return ( ... ) }` and factor some out instead